portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,26 @@ function randomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = Fact;
 }
+function getName(){
+    
+     fetch('/data').then(response => response.json()).then((list) => {
+         console.log(list[0])
+    const json = document.getElementById('list-container');
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers): Give as descriptive a name as possible, within reason.

`json` does not reflect this variable well as it is an HTML element and not a json object being sent/received.